### ISSUE TITLE
kubectl.sh is deprecated

### ISSUE THREAD

**Issue by mvernimmen** [kubernetes/heapster#1330]
I'm new at using kubernetes and Heapster, so I followed Heapster's installation instructions for use with InfluxDB (https://github.com/kubernetes/heapster/blob/master/docs/influxdb.md). It states
`Ensure that kubecfg.sh is exported`
But according to https://github.com/kubernetes/kubernetes/issues/2573 kubecfg.sh is deprecated and should be replaced with kubectl (not kubectl.sh). 

In my installed version of kubernetes 1.4 (through kubeadm), there is no kubecfg.sh to be found anywhere.

Proceeding with the next step despite this missing component results in this:
```
[root@td100 k8]# kubectl create -f deploy/kube-config/influxdb/
the path "deploy/kube-config/influxdb/" does not exist
```
Maybe obvious, but not to all, one needs to either clone the repo first:
```
[root@td100 k8]# git clone https://github.com/kubernetes/heapster.git
Cloning into 'heapster'...
remote: Counting objects: 22542, done.
remote: Total 22542 (delta 0), reused 0 (delta 0), pack-reused 22541
Receiving objects: 100% (22542/22542), 34.13 MiB | 8.59 MiB/s, done.
Resolving deltas: 100% (11060/11060), done.
[root@td100 k8]# cd heapster/
[root@td100 heapster]# kubectl create -f deploy/kube-config/influxdb/
service "monitoring-grafana" created
replicationcontroller "heapster" created
service "heapster" created
replicationcontroller "influxdb-grafana" created
service "monitoring-influxdb" created
[root@td100 heapster]#
```
or perhaps by using a full url to the right yaml on github (not sure which one to use, perhaps https://raw.githubusercontent.com/kubernetes/heapster/master/deploy/kube-config/influxdb/influxdb-service.yaml ?). After doing that installation, the graphs appear fine in the webui (so the kubecfg.sh really isn't needed).

In summary, the documentation referenced above is IMHO not correct (enough). Could it please be updated for others looking to add graphs to their kubernetes webui installation?
